_pages/en_US/dumping-cartridges.md

```diff
@@ -5,20 +5,24 @@ redirect_from:
   - /dump-card
 ---
 
-This is an add-on section for dumping cartridges using GodMode9i, useful for loading the cartridge on emulators, flashcarts or your SD card via nds-bootstrap.
+This is an add-on section for dumping cartridges using GodMode9i, useful for loading the cartridge data on emulators, flashcarts or your SD card via nds-bootstrap.
 
-### What You Need
-
-* Your game cartridge
+## Requirements
+* The cartridge of the title you would like to dump
 * Your Nintendo DSi console with [Unlaunch](installing-unlaunch) installed
-* The latest release of [GodMode9i](https://github.com/RocketRobz/godmode9i/releases/latest)
 
-### Instructions
+## Instructions
+### Section I - SD card Setup
+
+- Download the latest release of [GodMode9i](https://github.com/RocketRobz/godmode9i/releases/latest)
+- Extract `GodMode9i.nds` from the GodMode9i archive and place it anywhere on your SD card
+
+### Section II - Nintendo DSi Instructions
 
-1. Insert your game cartridge into your console
-1. Extract `GodMode9i.nds` from the GodMode9i archive and place it anywhere on your SD card
-1. Launch GodMode9i via the Unlaunch menu
-  - Hold **(A)** + **(B)** to go to the Unlaunch menu
+1. Power on your console while holding **(A)** and **(B)**
+   - This should launch the Unlaunch Management Menu
+1. Ensure the game cartridge is inserted into the console
+1. Launch GodMode9i in the Unlaunch Menu
```

Comment:
I would switch this and the line above as this implies that the cartridge has to be in before launching GM9i (and that you might not be able to do multiple without rebooting)